templates/admin/site_settings/index.html.twig

```diff
@@ -15,16 +15,23 @@
                     name="name[{{ item.id }}]"
                     class="form-control"
                     readonly="readonly"
-                    value="{{ aliases[item.name] }}"
+                    value="{{ aliases[item.name]['alias'] }}"
                     >
 
             </td>
             <td>
                 <input
-                    type="text"
+{#                    type="text"#}
```

Comment:
If we do not need this line just delete it.

Reply:
Done